src/services/tasks/types.ts

```diff
@@ -111,6 +120,30 @@ export interface TasksService {
    */
   cancel(taskId: ObjectId, options?: OptionsBase): Promise<AffectedRecords>;
 
+  /**
+   * ## Execute a Function directly
+   *
+   * A Function may be executed directly, the function will be executed synchronously and the response may be awaited by the caller.
```

Comment:
Enough detail? 🤔 Reference to documentation? 🤔

Reply:
All jokes aside, yes, clear enough in my opinion 👌